indico/modules/events/abstracts/controllers/reviewing.py

```diff
@@ -77,10 +77,13 @@ def _checkParams(self, params):
     def _process(self):
         query = (Abstract
                  .query.with_parent(self.event_new)
+                 .filter(Abstract.state.notin_({AbstractState.duplicate, AbstractState.merged}))
```

Comment:
I think they should be both duplicate of C, right? Especially if B has already been marked as duplicate of it.